flow-typed/module_vx.x.x.js

```diff
@@ -1,3 +1,3 @@
-declare module 'module' {
+declare module 'grpc' {
```

Comment:
We should probably create a new file for this rather than modifying the existing flowtype for `module`

Reply:
going to remove this as this was fixed in a diff PR and merged